.github/workflows/gradle-build.yml

```diff
@@ -11,7 +11,7 @@ permissions:
   contents: read
 
 jobs:
-  build:
```

Comment:
Could we keep the job name? This way we don't have to update the policy.

Reply:
I kept it but changed it to be an combination of the result of both build jobs: https://github.com/microsoftgraph/msgraph-beta-sdk-java/pull/988/files#diff-f40eb8ed4938a938963faf4e9f9f40cf7f965728b47bbac3653c6b39fc88a58eR69

Reply:
Sorry I had missed that part. Thank you for letting me know.